lib/api/contacts.js

```diff
@@ -3,3 +3,71 @@ export const queryEmails = (data) => ({
     method: 'get',
     data
 });
+
+export const getContacts = ({ Page = 0, PageSize = 1000, LabelID = '' }) => ({
+    url: `contacts/?${PageID},${PageSize},${LabelID}`,
```

Comment:
Don't format yourself queryParms, you can use ` params: { PageID }` etc. ;)

Reply:
Sorry I don't understand how to implement the suggestion. Could you give me more details?